README.md

```diff
@@ -13,6 +13,32 @@ This project is currently built with and tested against Hive 2.3.0 (and minor ve
 # Usage
 The BeeJU JUnit rules provide a way to run tests that have an underlying requirement to use the Hive Metastore API but don't have the ability to mock the [Hive Metastore Client](https://hive.apache.org/javadocs/r1.2.1/api/org/apache/hadoop/hive/metastore/HiveMetaStoreClient.html). The rules spin up and tear down an in-memory Metastore which may add few seconds to the test life cycle so if you require tests to run in the sub-second range this is not for you.
 
+##Maven Dependencies
+Depend on Beeju using:
+
+    <dependency>
```

Comment:
I think you can mark this as an XML code block, I know @AnanaMJ did something similar on code example sections for Circus Train or Waggle Dance (I think for YAML or Java) so take a look at their README.md and do something similar.

Reply:
yep, so:
```
"```xml
<dependency>
  <groupId>com.hotels</groupId>
  <artifactId>beeju</artifactId>
  <version>....</version>
  <scope>test</scope>
</dependency>
```"
```
with no start and end quotes, will look like this

```xml
<dependency>
  <groupId>com.hotels</groupId>
  <artifactId>beeju</artifactId>
  <version>....</version>
  <scope>test</scope>
</dependency>
```